cmd/blockchaincmd/create.go

```diff
@@ -222,7 +223,28 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	sc := &models.Sidecar{}
+
+	if err = promptValidatorManagementType(app, sc); err != nil {
+		return err
+	}
+
+	if sc.ValidatorManagement != models.ProofOfAuthority && createFlags.poaValidatorManagerOwner != "" {
+		return errors.New("--poa-manager-owner flag cannot be used when blockchain validator management type is not Proof of Authority")
+	}
+
 	if vmType == models.SubnetEvm {
+		if sc.ValidatorManagement == models.ProofOfAuthority {
```

Comment:
nit: we use it so many time we should add a func to sidecar `sc.PoA` returning bool and `sc.PoS` with this comparison